docusaurus.config.js

```diff
@@ -32,7 +32,8 @@ const config = {
     description: 'An open-source framework for building practical distributed replication mechanisms.',
     gitHubUrl: gitHubUrl,
     siteLicense: siteLicense,
-    recentBlogPostsOnHomePage: 5
+    recentBlogPostsOnHomePage: 5,
+    shortname: 'https-replica-io-dev'
```

Comment:
Once we're ready to merge this PR, we should change the value to `replica-io`.

Reply:
yups